bokehjs/src/coffee/core/selection_manager.ts

```diff
@@ -28,8 +28,8 @@ export class SelectionManager extends HasProps {
     this.inspectors = {}
   }
 
-  select(renderer_views: RendererView[], geometry: Geometry, final: boolean, append: boolean = false): boolean {
-    // divide renderers into glyph_renderers or graph_renderers
+  _split_renderer_views_by_type(renderer_views: RendererView[]): {[key: string]: RendererView[]} {
+    // TODO: return type, object values should be array of one type of renderer view
```

Comment:
You should also return an interface instead of a mapping, unless you can't (or won't be in future) predict what keys are returned.